react/src/components/UserProfileSettingModal.tsx

```diff
@@ -0,0 +1,304 @@
+import React from "react";
+import { Modal, Input, Form, Select, SelectProps, message, Switch } from "antd";
+import { useTranslation } from "react-i18next";
+import { useWebComponentInfo } from "./DefaultProviders";
+import { passwordPattern } from "./ResetPasswordRequired";
+import { useSuspendedBackendaiClient } from "../hooks";
+import { useTanQuery, useTanMutation } from "../hooks/reactQueryAlias";
+import _ from "lodash";
+
+const UserProfileSettingModal: React.FC = () => {
+  const { t } = useTranslation();
+
+  const [form] = Form.useForm();
+
+  const [messageApi, contextHolder] = message.useMessage();
+  const _showSuccessMessage = (successMessage: string) => {
+    messageApi.open({
+      type: "success",
+      content: successMessage,
+    });
+  };
+  const _showErrorMessage = (errorMessage: string) => {
```

Comment:
Please use `messageAPI.open` directly where it is needed. You don't need to define it as a separate function when the code is used more than once. It's better when the code inside the function is simple and clear.